resources/proxytrack.rb

```diff
@@ -0,0 +1,75 @@
+resource_name :proxytrack
+
+default_action :create
+
+property :proxy_address,      String
+property :proxy_port,         Integer
+property :icp_address,        String, default: 'localhost'
+property :icp_port,           Integer
+property :httrack_file_paths, Array
+
+action :create do
+  run_context.include_recipe 'proxytrack::default'
+
+  systemd_unit "proxytrack-#{new_resource.name}.service" do
+    content <<-EOU.gsub(/^\s+/, '')
```

Comment:
I generally prefer to use ``EOF``.
```suggestion
    content <<-EOF.gsub(/^\s+/, '')
```